Modules/OrgUnit/interfaces/class.ilOrgUnitUserQueriesInterface.php

```diff
@@ -22,4 +22,12 @@ public function findAllUsersByUserIds($user_ids);
 	 */
 	public function getAllUserNames($users);
 
+
+	/**
+	 * @param int $id $id user id
+	 *
+	 * @return array | null
+	 */
+	public function findUserById($id);
```

Comment:
Please rename this method to what it exactly does, something like `returnUserDataAsArrayForUserId()`.